services/migrations/gitea_uploader.go

```diff
@@ -217,18 +218,19 @@ func (g *GiteaLocalUploader) CreateMilestones(milestones ...*base.Milestone) err
 // CreateLabels creates labels
 func (g *GiteaLocalUploader) CreateLabels(labels ...*base.Label) error {
 	lbs := make([]*issues_model.Label, 0, len(labels))
-	for _, label := range labels {
+	for _, l := range labels {
 		// We must validate color here:
-		if !issues_model.LabelColorPattern.MatchString("#" + label.Color) {
-			log.Warn("Invalid label color: #%s for label: %s in migration to %s/%s", label.Color, label.Name, g.repoOwner, g.repoName)
-			label.Color = "ffffff"
+		if !label.ColorPattern.MatchString("#" + l.Color) {
+			log.Warn("Invalid label color: #%s for label: %s in migration to %s/%s", l.Color, l.Name, g.repoOwner, g.repoName)
+			l.Color = "ffffff"
 		}
 
 		lbs = append(lbs, &issues_model.Label{
 			RepoID:      g.repo.ID,
-			Name:        label.Name,
-			Description: label.Description,
-			Color:       "#" + label.Color,
+			Name:        l.Name,
+			Exclusive:   l.Exclusive,
+			Description: l.Description,
+			Color:       "#" + l.Color,
```

Comment:
`#RGB` will pass the ColorPattern check, and be used as `label.Color`, but it's not a valid color (not normalized), will break functions like ColorRGB/UseLightTextColor, etc.


IMO, the `ColorPattern` should be removed, only use `color, err := NormalizeColor`

Reply:
@wxiaoguang done